packages/viewer/src/components/viewer/viewer.tsx

```diff
@@ -128,7 +132,7 @@ export class Viewer {
    *
    * @private
```

Comment:
While you're in here, can you add `@internal` here so it doesn't show up in our Stencil docs?

Reply:
2e90a72